modules/virt-removing-interface-from-nodes.adoc

```diff
@@ -5,8 +5,10 @@
 [id="virt-removing-interface-from-nodes_{context}"]
 = Removing an interface from nodes
 
-Remove an interface from nodes by editing the `NodeNetworkConfigurationPolicy` object and set
-the `state` of the interface to `absent`.
+Remove an interface from nodes by editing the `NodeNetworkConfigurationPolicy` object and set the `state` of the interface to `absent`.
```

Comment:
IIUC from the example later in the text, it's supposed to be :
```
Remove an interface from nodes by editing the `NodeNetworkConfigurationPolicy` object by setting the `state` of the interface to `absent`
```

Also, I think we should rephrase this sentence, as both this paragraph and the following one start with the same "Remove an interface"/"Removing an interface".

Reply:
Thanks Ram, updated mostly as suggested